tests/integration/test_reshape.py

```diff
@@ -102,23 +103,36 @@ def test_ravel(self):
         assert np.array_equal(a.ravel(), anp.ravel())
 
 
-RECT_CASES = [
+RECT_CASES = (
     (10, 2, 10),
     (20, 10),
     (20, -5),
     (5, 40),
     (200, 1),
     (1, 200),
     (10, 20),
-]
+)
 
 
 class TestRect:
 
     anp = np.random.rand(5, 4, 10)
 
+    @pytest.mark.parametrize("order", ("C", "F", "A", None), ids=str)
     @pytest.mark.parametrize("shape", RECT_CASES, ids=str)
-    def test_shape(self, shape):
+    def test_shape(self, shape, order):
+        a = num.array(self.anp)
+        assert np.array_equal(
+            num.reshape(a, shape, order=order),
+            np.reshape(self.anp, shape, order=order),
+        )
+
+    @pytest.mark.xfail
+    @pytest.mark.parametrize("shape", (200, -1, -2, None), ids=str)
+    def test_0d(self, shape):
+        # for shape=None,
+        # In Numpy, pass, returns the flattened 1-D array
+        # In cuNumeric, raises TypeError: 'NoneType' object is not iterable
```

Comment:
Same comment as below. The scope of `xfail` tests must be as narrow as possible, and should not be mixed in with things that are passing. Otherwise the passing cases are reported a r “unexpectedly passing” (X) in the test output which is unnecessary and confusing

Reply:
Fixed.